tempodb/backend/s3/s3.go

```diff
@@ -343,6 +343,10 @@ func createCore(cfg *Config, hedge bool) (*minio.Core, error) {
 		return nil, errors.Wrap(err, "create minio.DefaultTransport")
 	}
 
+	if cfg.InsecureSkipVerify {
```

Comment:
i believe this will work but may be worth testing locally. you could probably spin up a minio with https and set this parameter pretty easily.

Insecure would need to be false.

Reply:
Alright, I've been able to test this and confirm that it does work.